.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .DS_Store*
 .hubot_history
+*.sw[o|p]
```

Comment:
ok, thought so. i would recommend using :set noswapfile in vim. swapfiles are stupid.